src/Components/pokemon/Pokemon.js

```diff
@@ -0,0 +1,48 @@
+import React, {useEffect, useState} from 'react';
+import axios from "axios";
+import "./Pokemon.css";
+
+const Pokemon =({endpoint}) => {
+
+    const [pokemonData, setPokemonData] = useState({});
```

Comment:
Hier ook, maar dan meer om makkelijker te kunnen checken of het nog ingeladen moet worden of niet
```suggestion
    const [pokemonData, setPokemonData] = useState();
```